code/modules/clothing/suits/jobs.dm

```diff
@@ -25,7 +25,7 @@
 		/obj/item/reagent_containers/spray/plantbgone,
 		/obj/item/secateurs,
 		/obj/item/seeds,
-		/obj/item/storage/bag/plants,
+		/obj/item/storage/bag/plants
```

Comment:
Why remove the trailing comma?

Reply:
I thought those were bad? i'll add them back in

Reply:
trailing commas good we love trailing commas

Reply:
okay apparently trailing commas are GOOD